lib/backpex/live_resource.ex

```diff
@@ -1585,35 +1590,35 @@ defmodule Backpex.LiveResource do
     ]
   end
 
-  def maybe_put_empty_filter(%{} = filters, empty_filter_key) when filters == %{} do
+  defp maybe_put_empty_filter(%{} = filters, empty_filter_key) when filters == %{} do
     Map.put(filters, Atom.to_string(empty_filter_key), true)
   end
 
-  def maybe_put_empty_filter(filters, _empty_filter_key) do
+  defp maybe_put_empty_filter(filters, _empty_filter_key) do
     filters
   end
 
   @doc """
   Returns list of filter options from query options
   """
-  def get_filter_options(module, query_options) do
+  def get_filter_options(query_options) do
     query_options
     |> Map.get(:filters, %{})
-    |> Map.drop([Atom.to_string(module.get_empty_filter_key())])
+    |> Map.drop([Atom.to_string(get_empty_filter_key())])
   end
 
   @doc """
   Returns list of active filters.
   """
-  def get_active_filters(module, assigns) do
-    filters = module.filters(assigns)
+  def active_filters(assigns) do
```

Comment:
```suggestion
  def active_filters(live_resource, assigns) do
```

Even if it's already included in the assigns, adding the `live_resource` as a param makes it more explicit.

I am unsure if I like it or not...

Reply:
I think its in line with the rest of the application if we just use the one from `assigns`.